test/integration/integration_test_fixtures.py

```diff
@@ -134,6 +140,15 @@ def tearDown(self):
         any_failed = True
     assert (not any_failed)
 
+    messages = []
```

Comment:
Maybe this name could be slightly clearer about what kind of messages they are? Probably not necessary.

Reply:
Addressed in 61821dbaeca4c7cddf6923cc38f0f693b760ca37.